src/I18n/Number.php

```diff
@@ -263,6 +284,31 @@ public static function defaultCurrency($currency = null)
         return self::$_defaultCurrency;
     }
 
+    /**
+     * Getter/setter for default currency format
+     *
+     * @param null $currencyFormat Default currency format to be used by currency()
+     * if $currencyFormat argument is not provided. If boolean false is passed, it will clear the
+     * currently stored value
+     * @return string CurrencyFormat
+     */
+    public static function defaultCurrencyFormat($currencyFormat = null)
+    {
+        if (!empty($currencyFormat)) {
```

Comment:
!empty() is bad here if format can also be false (next line). Use better strict `!== null` here

Reply:
But if it's `false`, it *does* need to continue on.

Reply:
Exactly, and it would with strict check.

Reply:
`false` is `!== null`, so this check would be true, and it would return right away. I can check for both `!== null` and `!== false`.